scene/gui/rich_text_label.cpp

```diff
@@ -6767,9 +6767,22 @@ void RichTextLabel::_apply_translation() {
 
 	internal_stack_editing = true;
 
+	// Infer closing tags for `bbcode_prefix` by reversing their order and removing parameters.
+	// Also use this to reconstruct a valid, matching BBCode prefix.
+	const PackedStringArray tags = bbcode_prefix.replace("[", "").split("]");
+	String valid_bbcode_prefix;
+	String bbcode_suffix;
+	for (const String &tag : tags) {
+		if (!tag.is_empty()) {
+			valid_bbcode_prefix += "[" + tag + "]";
+			bbcode_suffix = "[/" + tag.get_slice("=", 0) + "]" + bbcode_suffix;
```

Comment:
Some tags can be in `[tag option1=value1 option2=value2]` form, e.g., using `[font slant=0.3 emb=1.0]` will result in:

<img width="553" alt="Screenshot 2025-05-23 at 08 27 09" src="https://github.com/user-attachments/assets/c67c593f-6083-4a1d-831c-eba585902873" />

Reply:
Should be fixed now.